### PR TITLE
Add statesync example script

### LINKED ISSUE #415: Add statesync example script
like https://github.com/cosmos/gaia/blob/main/contrib/statesync.bash or https://github.com/osmosis-labs/osmosis/blob/main/scripts/statesync.sh

### PR DESCRIPTION
**Pull request by pinosu**
Resolves #415

### CODE REVIEW

File: scripts/statesync.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+LATEST_HEIGHT=$(curl -s $SNAP_RPC/block | jq -r .result.block.header.height);
+BLOCK_HEIGHT=$(($LATEST_HEIGHT-$INTERVAL))
+TRUST_HASH=$(curl -s "$SNAP_RPC/block?height=$BLOCK_HEIGHT" | jq -r .result.block_id.hash)
+
+# Print out block and transaction hash from which to sync state.
+echo "TRUST HEIGHT: $BLOCK_HEIGHT"
+echo "TRUST HASH: $TRUST_HASH"
+
+# Export state sync variables.
+export TGRADE_P2P_MAX_NUM_OUTBOUND_PEERS=200
```
Comment (alpe) [inline scripts/statesync.sh:35]: This is a node config that is not needed here

File: scripts/statesync.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+export TGRADE_STATESYNC_ENABLE=true
+export TGRADE_STATESYNC_RPC_SERVERS="$SNAP_RPC,$SNAP_RPC"
```
Comment (alpe) [inline scripts/statesync.sh:37]: Please use "$SNAP_RPC,https://tgrade-mainnet-rpc.allthatnode.com:26657" so that we have 2 independent sources

**Review by alpe — CHANGES_REQUESTED** [review]
Good start! I would strip it down to state sync only.

File: scripts/statesync.sh [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+export TGRADE_STATESYNC_TRUST_HEIGHT=$BLOCK_HEIGHT
+export TGRADE_STATESYNC_TRUST_HASH=$TRUST_HASH
```
[the remark this answers is not in the document]
Comment (pinosu) [reply scripts/statesync.sh:24]: I tested it multiple times and it works fine on my machine.

File: scripts/statesync.sh [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+# Print every command.
+set -ux
+
+# Initialize chain.
+tgrade init test --chain-id tgrade-mainnet-1
+
+# Get "trust_hash" and "trust_height".
+INTERVAL=2000
+SNAP_RPC="https://rpc.mainnet-1.tgrade.confio.run:443"
+LATEST_HEIGHT=$(curl -s $SNAP_RPC/block | jq -r .result.block.header.height);
```
Comment (maurolacy) [inline scripts/statesync.sh:12]:
```suggestion
LATEST_HEIGHT=$(curl -s $SNAP_RPC/block | jq -r .result.block.header.height)
```

**Review by maurolacy — APPROVED** [review]
LGTM.

**Review by alpe — APPROVED** [review]

**Pull request merged**